iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -233,32 +239,43 @@ public enum ErrorCode
         DeviceMessageLockLost = 412002,
 
         // RequestEntityTooLarge - 413
+
         /// <summary>
         /// When the message is too large for IoT hub you will receive this error. You should attempt to reduce your message size and send again.
-        /// For more information on message sizes, see <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-quotas-throttling#other-limits">IoT hub quotas and throttling | Other limits</see>
+        /// For more information on message sizes, see <see href="https://aka.ms/iothubthrottling#other-limits">IoT hub quotas and throttling | Other limits</see>
```

Comment:
Using the aka consistently to save characters.

Reply:
I checked to make sure section references (#other-limits) works when using the aka.